idd/Energy+.idd.in

```diff
@@ -42533,10 +42533,8 @@ Coil:Cooling:DX:VariableSpeed,
         \type alpha
         \reference CoolingCoilsDXVariableSpeed
    A2,  \field Indoor Air Inlet Node Name
-        \required-field
```

Comment:
Hmm, implications of removing required-field for all variable speed DX coil objects?  Maybe this will be resolved as I make my way through the rest of the PR review, just spewing stream of consciousness as I go.  

FYI @rraustad @mjwitte @EnergyArchmage